homer0/parserror
# Add ES modules version

### What does this PR do?

[Update with the reasons why I'm migration].

### How should it be tested manually?

Using Node >= 14:

```bash
echo "import { Parserror } from 'parserror/esm/lib.js';\n\nconsole.log(Parserror);" > test.mjs;
node ./test.mjs
```

You should see the log for the `Parserror` class.

And don't forget to...

```bash
yarn test
# or
npm test
```
